docs/UserGuide.md

```diff
@@ -35,100 +39,194 @@
 > * Items with `...` after them can have multiple instances.
 > * The order of parameters is fixed.
 
-#### Viewing help : `help`
-Format: `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
- 
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Changing storage location : `change`
+Changes the data file storage location. <br>
+Format: `change location to PATH`
+
+Examples:
+* `change location to C:\Users\Verbena\Dropbox\WhatNow`<br>
+Changes the data file storage location to C:\Users\Verbena\Dropbox\WhatNow<br>
+<img src="images/UI_Prototype/WhatNowUI-ChangeLocation-Colour.png" width="600">
+
+
+#### Adding a task: `add`
+Adds a task to WhatNow<br>
+Format: `add "DESCRIPTION" [on/by] [today/tomorrow/DAY/DATE] [from/at] [START_TIME] [to/till] [END_TIME] [every] [DAY/day/week/month/year] [low/medium/high]` 
```

Comment:
DAY in uppercase means it's a parameter as mentioned above so what it means is the user can type every Wednesday, every day, every week, every month or every year.